<?xml version="1.0"?>

<launch>
  <arg name="frame_follower"/>
  <arg name="frame_leader"/>
  <arg name="worldFrame" default="world"/>

  <node name="leader_follower" pkg="crazyflie_controller" type="leader_follower" output="screen">
    <param name="frame_follower" value="$(arg frame_follower)" />
    <param name="frame_leader" value="$(arg frame_leader)" />
    <param name="worldFrame" value="$(arg worldFrame)" />
    <rosparam command="load" file="$(find crazyflie_controller)/config/PID_values.yaml" />
  </node>
</launch>
